CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on the [KeepAChangeLog] project.
 - [#679] Make `state` optional in `EndSessionRequest`
 - [#683] Fix basic_auth with client password
 - [#698] `state` in `EndSessionRequest` request args and kwargs different
+- [#708] Wants the original non-parsed JWT and not an IDToken instance.
```

Comment:
This needs to go to the unreleased section as it is going to be a bugfix.

Reply:
Done